src/explorer.ts

```diff
@@ -188,6 +189,10 @@ class KubernetesResource implements KubernetesObject, ResourceNode {
         this.resourceId = kind.abbreviation + '/' + id;
     }
 
+    namespace(): string | null {
```

Comment:
can we turn this into a getter? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get